### DISCUSSION TITLE
problem with Next.js tutorial

### DISCUSSION THREAD

**Discussion by kilted2000** [vercel/next.js#50805] [Help]
### Summary

When I was going through the official Next.js tutorial I did everything I was told to do and it is all working. But when I move between pages the background does not persist as it should. This is the page I am talking about. I am using vs code and chrome.

Client-Side Navigation
The [Link](https://nextjs.org/docs/api-reference/next/link) component enables client-side navigation between two pages in the same Next.js app.

Client-side navigation means that the page transition happens using JavaScript, which is faster than the default navigation done by the browser.

Here’s a simple way you can verify it:

Use the browser’s developer tools to change the background CSS property of <html> to yellow.
Click on the links to go back and forth between the two pages.
You’ll see that the yellow background persists between page transitions.
This shows that the browser does not load the full page and client-side navigation is working.
****When I do the above the background color changes to yellow but when I click between links the full page does load so the background goes back to white. Did I do something wrong even though everything else works as expected or is there nothing to worry about?****

If you’ve used <a href="…"> instead of <Link href="…"> and did this, the background color will be cleared on link clicks because the browser does a full refresh.

### Example

https://nextjs.org/learn/basics/navigate-between-pages/client-side

**Comment by flyisland** [comment]
I encountered the same issue; client-side navigation is not working with the Link component
